src/OrleansAzureUtils/Storage/AzureTableDataManager.cs

```diff
@@ -467,57 +437,93 @@ public AzureTableDataManager(string tableName, string storageConnectionString, L
         /// </summary>
         /// <param name="predicate">Predicate function to use for querying the table and filtering the results.</param>
         /// <returns>Enumeration of entries in the table which match the query condition.</returns>
+#if !NETSTANDARD
+        //Microsoft.WindowsAzure.Storage.Table.Queryable is not supported in netstandard, so TableQuery with an Expression predicate 
+        // is not supported in netstandard, just keep this method in full dotnet
+        // for full dotnet users consistency
         public async Task<IEnumerable<Tuple<T, string>>> ReadTableEntriesAndEtagsAsync(Expression<Func<T, bool>> predicate)
```

Comment:
I'm not sure about this. This project is called AzureUtils and this class can be perfectly used as a DAO layer between the application and AzureTable